nexus-writer/src/nexus/run_parameters.rs

```diff
@@ -0,0 +1,72 @@
+use anyhow::{anyhow, Result};
+use chrono::{DateTime, Utc};
+use supermusr_streaming_types::{
+    ecs_6s4t_run_stop_generated::RunStop, ecs_pl72_run_start_generated::RunStart,
+};
+
+#[derive(Default, Debug)]
+pub(crate) struct RunStopParameters {
+    pub(crate) collect_until: u64,
+    pub(crate) last_modified: DateTime<Utc>,
```

Comment:
What is this used for?

Reply:
This is used in the run buffering feature. If messages are coming in slow, the run isn't flushed from cache until a set time has passed since it was last added to.